### DISCUSSION TITLE
Is it possible to use the script on an already configured system?

### DISCUSSION THREAD

**Discussion by Fuhrerbaumanki** [prasanthrangan/hyprdots#244] [Q&A]
I have KDE and i3wm installed on Arch Linux, do I have to reinstall Arch for the script to work, or can I still use it on an already running system?
 P.S.
 I read the description of the script, but I'm still wondering if the script will work from an already configured system

**Comment by jpx40** [comment]
Yes u can use the Script on an already configured system, i did too. But make sure to backup your configs just in case.

**Comment by EvanSunde** [comment]
> Yes u can use the Script on an already configured system, i did too. But make sure to backup your configs just in case.

I installed it but it broke my grub and couldn't boot into linux

**Reply by prasanthrangan** [reply]
are you using nvidia? as it enables the drm modeset in grub.
Please try using an arch iso boot usb to chroot into your arch drive and rebuild grub.

**Reply by EvanSunde** [reply]
No, its intel uhd graphics. And I wrote manual boot loading script for grub. Its shows no suitable video found, booting in blind mode. But its works. Will there be any problem if i leave it as it is

**Reply by prasanthrangan** [reply]
hmm, for intel it should only apply the theme and nothing else... also there should be a grub.bkp file created from your previous config... so you can try rebuilding grub from your earlier config to restore it!

**Reply by EvanSunde** [reply]
Right there is my previous config thanks. But just curious is it bad to leave the no suitable video mode found, booting in blind mode error in grub.

**Reply by prasanthrangan** [reply]
I dont think it will cause any issues as you are able to boot into linux, but it just feels right when it works as intended :)

**Reply by EvanSunde** [reply]
it seems grub broke because it was outdated, i reinstalled the new version and it worked with hyprland config file.
